.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "esnext": true
+}
```

Comment:
I don't know what we'd want here but we can add to this as we go along. If you want to add something specific, feel free!